apps/studio/components/interfaces/Account/Preferences/AccountDeletion.tsx

```diff
@@ -0,0 +1,30 @@
+import Panel from 'components/ui/Panel'
+import { useProfile } from 'lib/profile'
+import { AlertDescription_Shadcn_, AlertTitle_Shadcn_, Alert_Shadcn_ } from 'ui'
+import { CriticalIcon } from 'ui-patterns/Icons/StatusIcons'
+import { DeleteAccountButton } from './DeleteAccountButton'
+
+export const AccountDeletion = () => {
+  const { profile } = useProfile()
+
+  return (
```

Comment:
can we ensure the user has deleted all projects and orgs before we let them submit the ticket? (fine in a follow up PR). Less back and forth that way

Reply:
Yes, that's a great suggestion and it's something I have already been looking into. I need to discuss it with the Frontend team first then follow up with another PR.

Reply:
no worries i'll look into it 🙂